src/main/java/racingcar/domain/Position.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain;
+
+public class Position {
+    private int position;
+
+    public Position() {
+        this.position = 0;
+    }
+
+    public void increase() {
+        position ++;
```

Comment:
```suggestion
        position++;
```
이렇게 쓰는게 컨벤션으로 알고 잇어요~